documents/int256.go

```diff
@@ -0,0 +1,44 @@
+package documents
+
+import (
+	"math/big"
+
+	"github.com/centrifuge/go-centrifuge/errors"
+)
+
+// Int256 represents a signed 256 bit integer
+type Int256 struct {
+	v *big.Int
+}
+
+// NewInt256 creates a new Int256 given a string
+func NewInt256(n string) (*Int256, error) {
+	nn := new(big.Int)
+	nn.SetString(n, 10)
+	if !isValidInt256(*nn) {
+		return nil, errors.NewTypedError(ErrInvalidInt256, errors.New("value: %s", n))
+	}
+	return &Int256{nn}, nil
+}
+
+// Bytes returns the byte representation of this int256
+func (i *Int256) Bytes() []byte {
```

Comment:
What about the `FromBytes` since this will be needed to convert back and forth from p2p layer to model layer